src/transformers/commands/pt_to_tf.py

```diff
@@ -80,6 +81,51 @@ def register_subcommand(parser: ArgumentParser):
         )
         train_parser.set_defaults(func=convert_command_factory)
 
+    @staticmethod
+    def compare_pt_tf_models(pt_model, pt_input, tf_model, tf_input):
+        """
+        Compares the tf and the pt models, given their inputs, returning a tuple with the maximum observed difference
```

Comment:
```suggestion
        Compares the TensorFload and PyTorch models, given their inputs, returning a tuple with the maximum observed difference
```
No abbreviations in the doc pretty please :-)